nerdlets/groundskeeper-nerdlet/nrds.scss

```diff
@@ -0,0 +1,339 @@
+      }
+  
+      &:focus {
+        border-color: #005054;
+        box-shadow: 0 0 0 4px #70ccd245;
+      }
+    }
+  
+    &[class*="is-disabled"] {
+      background-color: #f4f5f5;
+      background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAOCAYAAAAvxDzwAAAAAXNSR0IArs4c6QAAAbBJREFUOBGtkj9Lw0AYxu+a/rciFOwUhJQW+wmcLFatg7g5Ci6OfgLBycmp4OLmopODi4j2Czi5SJE2KRKDVkJta0iraEvB87nglbQ4tNUMufd97n1/ee69EPLPD/0LT9O0yS/G9gilU6FAYFdRlKp3XKBpmmG71bpkhKQJY+Sj0+HmtsYCGoYRtJvNcwcmHDH2xEOPyIddGWO+z3b7DLCs6IG123AweMDzkRwC5lU17RSwNQHD/Ao0EllRZNkeCQiYp1QuH6NpXcAopXcSpdlZWX4VmuNQ1/WZdrebg/UH3NY+bsv5migCjAJ2hOFvCA21RZ8kLSeTyYbQ+AqdkJKmnaBpk8c4wo2E+aRSqTcnxwv7h9jfFjlq1JDfvxiPx1962k/gXAqK670Nxubwb11UKpUQ14qqmnPDcMxyOBBY+g3G6x2H1Wp1wrLtPBrTXOQPGnleQLjjCFwj5N5DaQbuTaENrg6Qi7VaLdKwrCs3tK+YUt0vSRnM7LlPH0h6QK5zaN2y8hj+vLsObg2/17uQSCQqbv23uO/HjsVi79PR6CrOey2K8cVHggsYBsZ7vgHrfaPHGc9IvwAAAABJRU5ErkJggg==");
+      background-size: 8px 6px;
+      box-shadow: 0 1px 0 #f4f5f5;
+  
+      &:hover,
+      &[class*="is-checked"]:checked,
+      &:not([class*="is-checked"]) {
+        background-color: #f4f5f5;
+        border-color: #d5d7d7;
+      }
+  
+      &:not([class*="is-checked"]) {
+        background-image: none;
+  
+        &:hover {
+          background-color: #f4f5f5;
+          border-color: #d5d7d7;
+        }
+      }
+    }
+  }
+  
+  // ==============================================================
+  // Radio Buttons
+  // ==============================================================
+  
+  [class*="Radio-input"],
+  input[type="radio"] {
+    background-color: #fff;
+    border-color: #d5d7d7;
+  
+    &:hover {
+      border-color: #8e9494;
+      background-color: inherit;
+      box-shadow: 0 1px 0 rgba(36, 37, 37, 0.05);
+    }
+  
+    &:focus {
+      border-color: #007e8a;
+      box-shadow: 0 0 0 4px #70ccd245;
+    }
+  }
+  
+  [class*="Radio-input"]:checked,
+  input[type="radio"]:checked {
+    background-color: #007e8a;
+    border: none;
+  
+    &:before {
+      width: 6px;
+      height: 6px;
+    }
+  
+    &:hover {
+      background-color: #007e8a;
+      box-shadow: 0 1px 0 rgba(36, 37, 37, 0.05);
+    }
+  
+    &:focus {
+      border: 1px solid #005054;
+      box-shadow: 0 0 0 4px #70ccd245;
+  
+      &:before {
+        left: 4px;
+        top: 4px;
+      }
+    }
+  }
+  
+  [class*="Radio-input"]:disabled,
+  input[type="radio"]:disabled,
+  [class*="Radio-input"].is-disabled {
+    border-color: #d5d7d7;
+    background-color: #f4f5f5;
+  
+    &:hover {
+      border-color: #d5d7d7;
+      background-color: #f4f5f5;
+    }
+  
+    &:checked {
+      border: 1px solid #d5d7d7;
+      background-color: #f4f5f5;
+  
+      &:before {
+        background-color: #d5d7d7;
+        top: 4px;
+        left: 4px;
+      }
+    }
+  }
+  
+  // ==============================================================
+  // Typography
+  // ==============================================================
```

Comment:
Same question as for checkbox - can we just use the standard NR1 styles for all of our typography?